kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -429,9 +434,6 @@ private void getAllTopicsAsync(CompletableFuture<Map<String, List<TopicName>>> t
                             topicMapFuture.completeExceptionally(e);
                             return;
                         }
-                        if (topicMapFuture.isDone()) {
-                            return;
-                        }
```

Comment:
Why remove this check? This check intended to skip the followed logic if the `topicMapFuture` was completed exceptionally.

Reply:
Maybe we should use "isCompletedExceptionally" to make it clearer?

> Why remove this check? This check intended to skip the followed logic if the `topicMapFuture` was completed exceptionally.

Maybe we should use "isCompletedExceptionally" to make it clearer?

Reply:
Agree.